custom_components/kumo/climate.py

```diff
@@ -138,31 +165,34 @@ def __init__(self, kumo_api):
             try:
                 setattr(self, "_%s" % prop, None)
             except AttributeError as err:
-                _LOGGER.debug("Kumo %s: Initializing attr %s error: %s",
-                              self._name, prop, str(err))
+                _LOGGER.debug(
+                    "Kumo %s: Initializing attr %s error: %s",
+                    self._name,
+                    prop,
+                    str(err),
+                )
 
         self._available = True
 
     def update(self):
-        """Called by HA to trigger a refresh of cached state."""
+        """Call HA to trigger a refresh of cached state."""
```

Comment:
I believe this comment (and the next one) are no longer correct.

Reply:
The error I received was "first line should be in imperative mood"

Reply:
Then it should be "Trigger a refresh of cached state." You have changed the sense of who calls whom.

Reply:
Yep, I get it now...sorry, wasn't even looking at the sentence, just changed the first words to get it through..

Reply:
fixed in https://github.com/dlarrick/hass-kumo/pull/19/commits/2fb891201a5a811c70d6ff9a90643e4fa678393d